dotnet/src/SemanticKernel.Core/Functions/SemanticFunction.cs

```diff
@@ -77,7 +77,7 @@ internal sealed class SemanticFunction : ISKFunction, IDisposable
             functionName: functionName,
             loggerFactory: loggerFactory
         );
-        func.SetAIConfiguration(functionConfig.PromptTemplateConfig.Completion);
+        func.SetAIConfiguration(functionConfig.PromptTemplateConfig.Models.FirstOrDefault<AIRequestSettings>());
```

Comment:
What if the result is `null` ?
If `SetAIConfiguration` don't support null use `First<AIRequestSettings>()` and it will throw if PremptTemplateConfig is misconfigured.

Reply:
Null is OK, `SetAIConfiguration` accepts null.